### PR TITLE
Fixed Enchanting And ChatFormat for Plugins

### PR DESCRIPTION
**Pull request by yungtechboy1**
Now enchanting works!

### CODE REVIEW

**Comment by File14** [comment]
@Pub4Game

**Comment by yungtechboy1** [comment]
Ooops... Well This will help Also XD

File: src/main/java/cn/nukkit/item/ItemDye.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
     public static final int MAGENTA = 2;
     public static final int LIGHT_BLUE = 3;
     public static final int YELLOW = 4;
+    public static final int LAPIS = 4;
```
Comment (Pub4Game) [inline src/main/java/cn/nukkit/item/ItemDye.java]: Lapis color != Yellow color

**Comment by Pub4Game** [comment]
ChatFormat? What? Why is it necessary?

**Comment by yungtechboy1** [comment]
That was supposed to be in a different Pull Request

**Comment by yungtechboy1** [comment]
Ill Resubmit it a Push

**Comment by Tee7even** [comment]
TextFormat was changed for a reason, it's now more functional and is more proper Java code. Plugins just have to be recompiled, no need for this.

**Pull request closed without merging**